src/sgame/sg_admin.cpp

```diff
@@ -3245,9 +3245,10 @@ bool G_admin_warn( gentity_t *ent )
 	if( ( found = G_ClientNumbersFromString( name, pids, MAX_CLIENTS ) ) != 1 )
 	{
 		count = G_MatchOnePlayer( pids, found, err, sizeof( err ) );
-		ADMP( va( "%s %s %s %s %s", QQ( "^3$1$: ^7$2t$\n $3$$4$" ), "warn", ( count > 0 ) ?  Quote( Substring( err, 0, count ) ) : Quote( err ),
-		                                      ( count > 0 ) ? Quote( Substring( err, count + 1, strlen( err ) ) ) : "",
-				                              ( count > 0 ) ? "" : "" ) );
+		ADMP( va( "%s %s %s",
+				  QQ( "^3$1$: ^7$2t$" ),
+				  "warn",
+				  ( count > 0 ) ? Quote( Substring( err, 0, count ) ) : Quote( err ) ) );
```

Comment:
reminder from the [old thread](https://github.com/Unvanquished/Unvanquished/pull/965/files#r84266908):

@mbasaglia on 20 Oct 2016 Member
> I'm not familiar with Substring(), is the new code really equivalent? I'd expect the old one to skip err[count]

@Kangz 13 days ago Member
> Yeah this one looks fishy, however this line is full of undefined behavior because Substring returns a pointer to an internal static buffer. I'd still be fine with this change even if we don't know if it is semantically equivalent, because it fixes an UB in rarely used code.